docs/advanced/kiali.adoc

```diff
@@ -0,0 +1,187 @@
+visualizations. It fetches ingress data (such as request tracing with Jaeger),
+the listing and data of the services, health indexes, and so on.
+
+Kiali runs as a service together with Istio, and does not require any changes
+to Istio or Openshift configuration (besides the ones required to install
+Istio).
+
+== How to install Kiali?
+
+A prerequisite for installing Kiali is that you must have OpenShift and Istio
+installed and configured.
+
+Install Kiali with the following commands:
+
+[source, bash]
+----
+# This command installs Kiali's configmaps
+curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-configmap.yaml | \
+   VERSION_LABEL=master envsubst | oc create -n istio-system -f -
+
+# This command installs Kiali's secrets
+curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-secrets.yaml | \
+   VERSION_LABEL=master envsubst | oc create -n istio-system -f -
+
+# And this command installs the Kiali service
+curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali.yaml | \
+   IMAGE_NAME=kiali/kiali \
+   IMAGE_VERSION=latest \
+   NAMESPACE=istio-system \
+   VERSION_LABEL=master \
+   VERBOSE_MODE=4 \ 
+   JAEGER_SERVICE=tracing envsubst | oc create -n istio-system -f -
+----
+
+Installing Kiali may take a minute or two. You can use the following commands
+to see if the service is running:
+
+[source, bash]
+----
+oc project istio-system
+oc get pods
+----
+
+You'll see a result that's something like this:
+
+----
+NAME                                        READY     STATUS      RESTARTS   AGE
+elasticsearch-0                             0/1       Running     0          1d
+grafana-6bb556d859-sr2kn                    0/1       Running     0          1d
+istio-citadel-d8b87b7fb-m6kr8               0/1       Running     0          1d
+istio-egressgateway-5c7847547d-l79bc        0/1       Running     0          1d
+istio-ingress-d4898f4d8-c45kz               0/1       Running     0          1d
+istio-ingressgateway-54ddb65d74-sznf9       0/1       Running     0          1d
+istio-pilot-58c7c644c7-l88d7                0/2       Running     0          1d
+istio-policy-586559fdd9-l88v9               0/2       Running     0          1d
+istio-sidecar-injector-6fc85fbd55-6r2hj     0/1       Running     0          1d
+istio-statsd-prom-bridge-6dbb7dcc7f-7qmbw   0/1       Running     0          1d
+istio-telemetry-7988677557-tz4sm            0/2       Running     0          1d
+jaeger-agent-lmhbb                          0/1       Running     0          0s
+jaeger-collector-68fd846775-wll4r           0/1       Running     0          1d
+jaeger-query-58f4655965-q8jjk               0/1       Running     0          1d
+kiali-7966d98797-77gqz                      0/1       Running     0          1d
+prometheus-586d95b8d9-r6d6x                 0/1       Running     0          1d
+----
+
+If the "STATUS" column for Kiali is "Running", you can check which URL it is
```

Comment:
We should wait until READY is 1/1. STATUS Running doesn't mean that is ready and people accessing the route sometimes can face the "Application is not available" message because the application is not ready. Example: https://github.com/redhat-developer-demos/istio-tutorial/blob/master/docs/2deploy-microservices.adoc#wait-preference-to-be-deployed